LEGO1/mxdsobject.cpp

```diff
@@ -113,16 +113,16 @@ unsigned int MxDSObject::CalculateUnk08()
 }
 
 // OFFSET: LEGO1 0x100bfa20
-void MxDSObject::Parse(char **p_source, MxU16 p_unk24)
+void MxDSObject::Parse(char **p_source, MxS16 p_unk24)
```

Comment:
I think chances are pretty good the source is actually a buffer/stream class/struct, with (all inline) members to extract data of specific types and advance the pointer. I have a difficult time believing anyone would manually do "get data, cast to correct type, advance pointer by data-type size" over and over when it could be placed in a tiny neat wrapper.
Just an idea.

Reply:
Good idea! I'd leave this for a separate PR though